lua/kickstart/plugins/debug.lua

```diff
@@ -89,6 +89,19 @@ return {
       },
     }
 
+    -- Change breakpoint icons
+    -- For more information, see https://github.com/mfussenegger/nvim-dap/discussions/355
+    vim.api.nvim_set_hl(0, 'Break', { fg = '#e51400' })
+    vim.api.nvim_set_hl(0, 'Stop', { fg = '#ffcc00' })
+    local breakpoint_icons = vim.g.have_nerd_font
+        and { Breakpoint = '', BreakpointCondition = '', BreakpointRejected = '', LogPoint = '', Stopped = '' }
```

Comment:
I think it be better to just have one table of icons

Reply:
Thank you for reviewing the code. With all due respect, that might not be possible.
If `vim.g.have_nerd_font` is `false`, the local variable breakpoint_icons would no longer exist.

Error that occurs when deleting the line:
```
Cannot assign `boolean|table` to parameter `<T:table>`.
   - `boolean` cannot match `table`
   - Type `boolean` cannot match `table` [param-type-mismatch]
```